Military Overlay Projects have updated GDB

Dbarnes1 opened the pull request:
Project, Project Template, Project Package, Layer Package have updated GDB schema for Military Overlay. In addition, I added the new standalone GDB zipped up.

Dbarnes1 commented:
Added commits for test datasets, now they are updated with new GDB

csmoore commented:
@Dbarnes1 - I should be able to review this PR tomorrow/Tues.

What does the added project `GDBNoSubtypes.ppkx` do?

Dbarnes1 commented:
This is another test dataset with sample data, similar to CertificationTest1, except the sample data are from layers that changed with the subtype to no subtype switch. If you only want to have the CertificationTest on the repository, just let me know

csmoore commented:
Only minor thing I noticed with the updated GDB is some featureclasses lacked a description (but oddly the Project layers had one)

![image](https://cloud.githubusercontent.com/assets/3090809/12905169/ebe6e850-cea2-11e5-8b72-46052396dd16.png)

![image](https://cloud.githubusercontent.com/assets/3090809/12905183/10ca0ff8-cea3-11e5-8292-31263a85b41c.png)

Also not sure where these extra feature classes came from in the MilitaryOverlay.aprx 
![image](https://cloud.githubusercontent.com/assets/3090809/12905225/69d6e4ae-cea3-11e5-8d74-101f78f9e16d.png)

csmoore commented:
Updated TOC layout looks good, but perhaps METOC Points, Lines, Areas should be grouped together (ex. MetocPointsAtmos, MetocPointsOceano, MetocLinesAtmos, MetocLinesOceano, etc.) and match the same Z-ordering of the ControlMeasure group (Point, Lines, then Areas) 

![image](https://cloud.githubusercontent.com/assets/3090809/12905289/e8600cf6-cea3-11e5-9783-94e24ccf3c8f.png)
![image](https://cloud.githubusercontent.com/assets/3090809/12905293/01f6feea-cea4-11e5-8d77-61a851df32a6.png)

csmoore commented:
@Dbarnes1 - I am done verifying/reviewing. Changes generally look good - a few last minor things I noticed while spot checking a few of the last projects.

Versions table does not need to be added to the map (noticed in MilitaryOverlay.ppkx):

![image](https://cloud.githubusercontent.com/assets/3090809/12905463/4cc14d8a-cea5-11e5-8f9a-3461c23abd8f.png)

CertificationTest1.ppkx:

![image](https://cloud.githubusercontent.com/assets/3090809/12905511/c1b5bb76-cea5-11e5-8542-d057e498161e.png)

You also may want to clear editor history before packaging (that used to be a recommendation, not sure if still the case):
![image](https://cloud.githubusercontent.com/assets/3090809/12905574/3a0a5726-cea6-11e5-9d0f-dd0ed1f822db.png)

Dbarnes1 commented:
Added metadata when missing in feature classes:
![image](https://cloud.githubusercontent.com/assets/13024357/12921916/3a71b16a-cf04-11e5-9dd0-a3219f6ac79b.png)

I also removed the extra feature classes, those were a test for the sde gdb and exporting to server. 

Reordered METOC layers in table of contents:
![image](https://cloud.githubusercontent.com/assets/13024357/12921973/7bc85402-cf04-11e5-9bc2-b96275fc14c7.png)

I also removed the versions table from the TOC and removed the geoprocessing history.
